appveyor.yml

```diff
@@ -1,13 +1,17 @@
 version: '{build}'
 
 environment:
-  mssql_jdbc_test_connection_properties: jdbc:sqlserver://localhost:1433;instanceName=SQL2016;databaseName=master;username=sa;password=Password12!;
+  JAVA_HOME: C:\Program Files\Java\jdk1.8.0 
+
   matrix:
-    - JAVA_HOME: C:\Program Files\Java\jdk1.8.0 
+    - mssql_jdbc_test_connection_properties: jdbc:sqlserver://localhost:1433;instanceName=SQL2008R2SP2;databaseName=master;username=sa;password=Password12!;
+
+    - mssql_jdbc_test_connection_properties: jdbc:sqlserver://localhost:1433;instanceName=SQL2016;databaseName=master;username=sa;password=Password12!;
 
 services:
+  - mssql2008r2sp2
```

Comment:
Hello @nsidhaye Hope you are doing well! from my investigation, matrix does not support services, I tested it in my personal fork and it worked this way. Thank you for sharing the link, I definitely will try it! If it works I will update my PR. Thank you :)